### DISCUSSION TITLE
Multiple Picker Field Selections

### DISCUSSION THREAD

**Discussion by ccgould** [enisn/UraniumUI#372] [Q&A]
How do I get the selections from the MultiplePickerField. I tried binding to the "SelectedItems" property with an "ObservableCollection" of object and it doesn't get any items I selected 

 `<material:MultiplePickerField Title="Pick additional options" SelectedItems="{Binding SelectedItems}" ItemsSource="{Binding AdditionalOptions}"/>
`

`public ObservableCollection<object> SelectedItems { get; set; }`

**Comment by enisn** [comment]
It should be updated, if not it's a bug, let me create an issue and check it
